tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -2965,6 +2973,55 @@ declare namespace ts {
                 bigintLiteral = 25
             }
         }
+        namespace typingsInstaller {
+            interface Log {
+                isEnabled(): boolean;
+                writeLine(text: string): void;
+            }
+            type RequestCompletedAction = (success: boolean) => void;
+            interface PendingRequest {
+                requestId: number;
+                packageNames: string[];
+                cwd: string;
+                onRequestCompleted: RequestCompletedAction;
+            }
+            abstract class TypingsInstaller {
+                protected readonly installTypingHost: InstallTypingHost;
+                private readonly globalCachePath;
+                private readonly safeListPath;
+                private readonly typesMapLocation;
+                private readonly throttleLimit;
+                protected readonly log: Log;
+                private readonly packageNameToTypingLocation;
+                private readonly missingTypingsSet;
+                private readonly knownCachesSet;
+                private readonly projectWatchers;
+                private safeList;
+                readonly pendingRunRequests: PendingRequest[];
```

Comment:
Technically nobody uses this publicly except our tests, and this member is the only reason we need to export a `PendingRequest`.

Reply:
we can't make it internal/private anymore, though, can we?

Reply:
Is there a reason you can't mark it as `/** @internal */`? The only methods that use it are private, and the tests are just checking for the length.

Reply:
I would just mark it as `@internal` which should fix things.

Reply:
done.